.github/workflows/gradle.yml

```diff
@@ -35,7 +35,16 @@ jobs:
           java-version: '11'
           java-package: jdk+fx
 
-      - name: Build and check with Gradle
+      - name: Build and check with Gradle (Ubuntu)
+        if: matrix.platform == 'ubuntu-latest'
```

Comment:
Thanks for adding GUI testing.

What's the difference between matrix context and runner context? https://stackoverflow.com/a/57948488 Since the other code above checks `runner.os == 'Linux'`.

Reply:
I'm not sure actually, but I'm changing it to `os` for consistency with the rest of the code.